src/python/pathops/operations.py

```diff
@@ -1,7 +1,7 @@
 from functools import partial
 from . import Path, PathOp, op
 
-__all__ = [op.name.lower() for op in PathOp]
+__all__ = [op_.name.lower() for op_ in PathOp]
```

Comment:
can you rename this `op_` simply `operation`?

Reply:
Yes, that works.